File: share/conf/vimspector/vimspector.json.c
{
"configurations": {
"linux debug": {
      "adapter": "vscode-cpptools",
     "configuration": {
        "request": "launch",
        "program": "/data/server/redis/bin/redis-server",
        "MIMode": "gdb",
        "stopAtEntry": true,
        "setupCommands": [
            {"text": "-gdb-set follow-fork-mode child"}
        ]
      }
     }
}}

File: share/conf/vimspector/vimspector.json.go
{
  "configurations": {
    "run": {
      "adapter": "vscode-go",
      "configuration": {
        "args": [ "*${args: -conf-dir conf -port 8888 -log-dir logs}" ],
        "request": "launch",
        "program": "${fileDirname}",
        "mode": "debug",
        "dlvToolPath": "$HOME/go/bin/dlv"     }
    }
  }
}

